src/commands/chains.ts

```diff
@@ -3,16 +3,38 @@ import { GlobalOptions } from "../cli.js";
 import { getChains } from "../utils.js";
 import type yargs from "yargs";
 
-export type Options = GlobalOptions;
+export interface Options extends GlobalOptions {
+  format: "pretty" | "json" | "jsonl";
```

Comment:
We use the `--format=` pattern elsewhere, so I used it here instead of `--pretty` so we stay consistent.